client/src/components/Search/SearchResultList/SearchResultList.tsx

```diff
@@ -14,14 +14,19 @@ interface SearchResultListProps {
   data: LocationInfo[] | null;
   loading?: boolean;
   error?: boolean | string | null;
+  handleSearchClear: () => void;
 }
 
-const SearchResultList = ({ data, loading, error }: SearchResultListProps) => (
+const SearchResultList = ({ data, loading, error, handleSearchClear }: SearchResultListProps) => (
   <S.SearchResultList>
     <RequestDataWrapper data={data} loading={loading} error={error}>
-      {data?.map(item => (
-        <SearchResultItem key={item.id} data={item} />
-      ))}
+      {data && data?.length > 0 ? (
```

Comment:
`!!data?.length` should be enough. otherwise `?` is not needed

Reply:
It does not work with `!!data?.length`, but you're right, as it is now, `?` is not needed, I will remove it.